packages/core/src/state-management/execution-queue.ts

```diff
@@ -0,0 +1,60 @@
+import { NamedTaskQueue } from './named-task-queue';
+import { DocID } from '@ceramicnetwork/docid';
+import { Task, TaskQueueLike } from '../pubsub/task-queue';
+import { DiagnosticsLogger } from '@ceramicnetwork/logger';
+import { RunningState } from './running-state';
+
+export interface ExecLike extends TaskQueueLike {
+  addE: (task: (state$: RunningState) => Promise<void>) => void;
```

Comment:
If you have an ExecutionQueue, when would you call `.addE` or `runE` vs `.add` or `.run`?  Some comments here would help

Reply:
Replaced `addE`/`runE` with just `add`/`run`. Sans-E functions are not used anyway.